src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -1857,6 +1874,16 @@ public void setReadTimeout(final int timeout) {
     this.readTimeout = timeout;
   }
 
+  /** Set a integrationName for all SDK requests. This method is only for test don't use it */
+  public void setIntegrationName(final IntegrationName integrationName) {
+    this.integrationName = integrationName;
+  }
+
+  /** Set a version for all SDK requests. This method is only for test don't use it */
+  public void setVersion(final String version) {
+    this.version = version;
+  }
+
```

Comment:
Make these package private if they are only for testing.

Reply:
I recommend making these fields `final` and rebuilding a ContrastSDK to change them instead of introducing this mutability just for testing.